app/javascript/App/components/NewsfeedPage/SearchCoins.js

```diff
@@ -34,21 +34,27 @@ class SearchCoins extends Component {
       updateUser,
       user,
       onWatch,
+      addCoinsToWatchlist,
+      rmCoinsWatchlist,
```

Comment:
Let's go ahead and stick with the fully expanded version `removeCoinsFromWatchlist`

Reply:
👍